### PR TITLE
Make the ugly fix slightly less ugly

### PR DESCRIPTION
**Pull request by monoidic**
To "fix" the "ugly hack" for #18
This will split sandbox-compat.patch into the actual patch itself and add the required files seperately. This should be cleaner and make updating the files seperately simpler.

### CODE REVIEW

File: app-crypt/qdigidoc/qdigidoc-3.13.6.ebuild [hunk 1]
```diff
@@ -39,6 +39,16 @@ DOCS="AUTHORS README.md"
 # gentoo specific zlib internal macro names
 append-cppflags "-DOF=_Z_OF"
 
+src_prepare() {
+	default
+	cp ${FILESDIR}/estonian-tsl.xml	${S}/client/EE.xml
+	cp ${FILESDIR}/tl-mp.xml	${S}/client/tl-mp.xml
+	cp ${FILESDIR}/TSL.qrc		${S}/client/TSL.qrc
+	cp ${FILESDIR}/config.json	${S}/common/config.json
```
Comment (metsma) [inline app-crypt/qdigidoc/qdigidoc-3.13.6.ebuild]: `# https://id.eesti.ee/config.json`

File: app-crypt/qdigidoc/qdigidoc-3.13.6.ebuild [hunk 2]
```diff
@@ -39,6 +39,16 @@ DOCS="AUTHORS README.md"
...
+	cp ${FILESDIR}/config.rsa	${S}/common/config.rsa
+	cp ${FILESDIR}/config.pub	${S}/common/config.pub
```
Comment (metsma) [inline app-crypt/qdigidoc/qdigidoc-3.13.6.ebuild]: `# https://id.eesti.ee/config.pub`

File: app-crypt/qdigidoc/qdigidoc-3.13.6.ebuild [hunk 3]
```diff
@@ -39,6 +39,16 @@ DOCS="AUTHORS README.md"
 # gentoo specific zlib internal macro names
 append-cppflags "-DOF=_Z_OF"
 
+src_prepare() {
+	default
+	cp ${FILESDIR}/estonian-tsl.xml	${S}/client/EE.xml
+	cp ${FILESDIR}/tl-mp.xml	${S}/client/tl-mp.xml
+	cp ${FILESDIR}/TSL.qrc		${S}/client/TSL.qrc
+	cp ${FILESDIR}/config.json	${S}/common/config.json
+	cp ${FILESDIR}/config.rsa	${S}/common/config.rsa
```
Comment (metsma) [inline app-crypt/qdigidoc/qdigidoc-3.13.6.ebuild]: `# https://id.eesti.ee/config.rsa`

File: app-crypt/qdigidoc/qdigidoc-3.13.6.ebuild [hunk 4]
```diff
@@ -39,6 +39,16 @@ DOCS="AUTHORS README.md"
 # gentoo specific zlib internal macro names
 append-cppflags "-DOF=_Z_OF"
 
+src_prepare() {
+	default
+	cp ${FILESDIR}/estonian-tsl.xml	${S}/client/EE.xml
+	cp ${FILESDIR}/tl-mp.xml	${S}/client/tl-mp.xml
```
Comment (metsma) [inline app-crypt/qdigidoc/qdigidoc-3.13.6.ebuild]: `# https://ec.europa.eu/information_society/policy/esignature/trusted-list/tl-mp.xml`

File: app-crypt/qdigidoc/qdigidoc-3.13.6.ebuild [hunk 5]
```diff
@@ -39,6 +39,16 @@ DOCS="AUTHORS README.md"
 # gentoo specific zlib internal macro names
 append-cppflags "-DOF=_Z_OF"
 
+src_prepare() {
+	default
+	cp ${FILESDIR}/estonian-tsl.xml	${S}/client/EE.xml
```
Comment (metsma) [inline app-crypt/qdigidoc/qdigidoc-3.13.6.ebuild]: `# https://sr.riik.ee/tsl/estonian-tsl.xml`

**Review by metsma — COMMENTED** [review]
Can you add links with comments to original file sources

**Comment by monoidic** [comment]
Alright. I'm not sure on a specific source for the .qrc though, unless linking https://github.com/open-eid/qdigidoc/wiki/DeveloperTips#building-in-sandboxed-environment or just leaving that blank is okay. Got any ideas?

**Comment by metsma** [comment]
You can leave blank or link to this article? Can you add comments before copy statement, then it seems more natural to read. And maybe qrc to first with this wiki link

**Review by metsma — APPROVED** [review]

**Comment by urmet** [comment]
Oh, nice. One tiny change though. In `src_prepare()` use `cmake-utils_src_prepare` instead of `default`.
With default you get an annoying warning ` * cmake-utils_src_prepare has not been run, please open a bug on https://bugs.gentoo.org/`

**Comment by urmet** [comment]
Thanks :)

**Pull request merged**